lib/app/router.js

```diff
@@ -2,20 +2,22 @@ import Vue from 'vue'
 import Router from 'vue-router'
 
 <% function recursiveRoutes(routes, tab, components) {
-  let res = ''
-  routes.forEach((route, i) => {
+  return routes.map((route, i) => {
```

Comment:
@manniL Don't really agree with this change. In fact, I think using `map()` crossed the original author's mind, but he stuck with `forEach()` for readability reasons. That's what I would have done too. Having a line finish with `join('')` looks a bit weird to me.

Other than that, I think all `reduce()`-based changes are great!

Reply:
@manniL upon further investigation, I do see the idiom used elsewhere in the case, so I'm assuming that is okay too :)

Reply:
Thanks for your feedback @galvez! 

It was my intention to encapsulate a single `resource` file and to join all of them together in the end, so possible debugging of a single resource object would be easier as the same operation is applied to all of them first. :relaxed: